mLRS/Common/hal/esp/tx-hal-radiomaster-nomad-esp32.h

```diff
@@ -0,0 +1,407 @@
+    ledRGB.SetPixelColor(0, RgbColor(0, 255, 0));
+    ledRGB.SetPixelColor(1, RgbColor(0, 255, 0));
+    ledRGB.Show();
+    ledGreenState = 1;
+}
+
+IRAM_ATTR void led_green_toggle(void)
+{
+    if (ledGreenState) { led_green_off(); } else { led_green_on(); }
+}
+
+IRAM_ATTR void led_blue_off(void)
+{
+    if (!ledBlueState) return;
+    ledRGB.SetPixelColor(0, RgbColor(0, 0, 0));
+    ledRGB.SetPixelColor(1, RgbColor(0, 0, 0));
+    ledRGB.Show();
+    ledBlueState = 0;
+}
+
+IRAM_ATTR void led_blue_on(void)
+{
+    if (ledBlueState) return;
+    ledRGB.SetPixelColor(0, RgbColor(0, 0, 255));
+    ledRGB.SetPixelColor(1, RgbColor(0, 0, 255));
+    ledRGB.Show();
+    ledBlueState = 1;
+}
+
+IRAM_ATTR void led_blue_toggle(void)
+{
+    if (ledBlueState) { led_blue_off(); } else { led_blue_on(); }
+}
+
+
+//-- Serial or Com Switch
+
+#ifdef DEVICE_HAS_SERIAL_OR_COM
+
+bool tx_ser_or_com_serial = true; // default is serial
+
+void ser_or_com_init(void)
+{
+    gpio_init(BUTTON, IO_MODE_INPUT_PU);
+    gpio_init(BUTTON2, IO_MODE_INPUT_PU);
+    uint8_t cnt = 0;
+    for (uint8_t i = 0; i < 16; i++) {
+        if (gpio_read_activelow(BUTTON) || gpio_read_activelow(BUTTON2)) cnt++;
+    }
+    tx_ser_or_com_serial = !(cnt > 8);
+}
+
+IRAM_ATTR bool ser_or_com_serial(void) { return tx_ser_or_com_serial; }
+
+IRAM_ATTR void ser_or_com_set_to_com(void) { tx_ser_or_com_serial = false; }
+
+#endif // DEVICE_HAS_SERIAL_OR_COM
+
+
+//-- Cooling Fan
+
+#define FAN_IO                    IO_P2
+
+void fan_init(void) { gpio_init(FAN_IO, IO_MODE_OUTPUT_PP_LOW); }
+
+IRAM_ATTR void fan_set_power(int8_t power_dbm)
+{
+    if (power_dbm >= POWER_23_DBM) { gpio_high(FAN_IO); } 
+    else { gpio_low(FAN_IO); }
+}
+
+
+//-- ESP32 Wifi Bridge
+
+#ifdef DEVICE_HAS_ESP_WIFI_BRIDGE_ON_SERIAL2  // this is an ESP32C3
+
+#define ESP_RESET                 IO_P19 // backpack_en
+#define ESP_GPIO0                 IO_P23 // backpack_boot
+#define ESP_BOOT0                 IO_P0  // will always be IO_P0
+
+uint8_t esp_boot0() { return gpio_read_activelow(ESP_BOOT0); }
+
+void esp_init(void)
+{
+    gpio_init(ESP_RESET, IO_MODE_OUTPUT_PP_LOW); // low -> esp is in reset
+    gpio_init(ESP_GPIO0, IO_MODE_OUTPUT_PP_LOW); // high -> esp will start in bootloader mode
+    // no need to configure ESP_BOOT0 which will always be IO_P0 and is pull-up by default
+}
+
+IRAM_ATTR void esp_reset_high(void) { gpio_high(ESP_RESET); }
+IRAM_ATTR void esp_reset_low(void) { gpio_low(ESP_RESET); }
+
+IRAM_ATTR void esp_gpio0_high(void) { gpio_low(ESP_GPIO0); }
+IRAM_ATTR void esp_gpio0_low(void) { gpio_high(ESP_GPIO0); }
+
+#endif // DEVICE_HAS_ESP_WIFI_BRIDGE_ON_SERIAL2 
+
+
+//-- POWER
+
+#include "../../setup_types.h" // needed for frequency band condition in rfpower calc
+#define SX_USE_LP_PA  // Nomad uses the low power amplifier for the 900 side
+
+
+void lr11xx_rfpower_calc(const int8_t power_dbm, uint8_t* sx_power, int8_t* actual_power_dbm, uint8_t frequencyBand)
```

Comment:
pl call the last variable frequency_band, and const it
see e.g. here https://github.com/olliw42/mLRS/blob/main/mLRS/Common/hal/hal-power-pa.h#L180